.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨
+
+Half of all issues filed here are duplicates, answered in the FAQ, or not appropriate for the bug tracker. Even if you think you've found a *bug*, please read the FAQ first, especially the Common "Bugs" That Aren't Bugs" section!
+
+Please help us by doing the following steps before logging an issue:
+  * Search: https://github.com/Microsoft/TypeScript/search?type=Issues
+  * Read the FAQ: https://github.com/Microsoft/TypeScript/wiki/FAQ
+
+Please fill in the *entire* template below.
+-->
+
+<!-- Please try to reproduce the issue with `typescript@next`. It may have already been fixed. -->
+**TypeScript Version:**  2.9.0-dev.201xxxxx
+
+<!-- Search terms you tried before logging this (so others can find this issue more easily) -->
+**Search Terms:** 
+
+**Code**
+
+```ts
+// A *self-contained* demonstration of the problem follows...
+// Test this by running `tsc` on the command-line, rather than through another build tool such as Gulp, Webpack, etc.
+```
+
+**Expected behavior:**
+
+**Actual behavior:**
+
+**Playground Link:** <!-- A link to a TypeScript Playground "Share" link which demonstrates this behavior --> 

```

Comment:
Maybe we should include an actual link to an empty playground rather than referencing "a TypeScript Playground".